ci/cloudbuild/builds/integration-nightly.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ATTENTION: This build is different than most because it's designed to be run
+# as a "manual" trigger that's run on a schedule in GCB. It will be scheduled
+# to right once a day at night. For details about scheduled manual triggers see
```

Comment:
"At night" is in the eye of the beholder.

Reply:
Just copying the existing name. Happy to rename to whatever you suggest.

Reply:
consider "once a day, at a time when most of the `google-cloud-cpp` developers are not working" or something like that.

Reply:
SGTM. Changed all references to "nightly" to "daily".